ports/apr-util/portfile.cmake

```
@@ -1,10 +1,18 @@
 include(vcpkg_common_functions)
 
-vcpkg_download_distfile(ARCHIVE
-  URLS "https://archive.apache.org/dist/apr/apr-util-1.6.0-win32-src.zip"
-  FILENAME "apr-util-1.6.0-win32-src.zip"
-  SHA512 98679ea181d3132020713481703bbefa0c174e0b2a0df65dfdd176e9771935e1f9455c4242bac19dded9414abe2b9d293fcc674ab16f96d8987bcf26346fce3a
-)
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
```

```suggestion
if(VCPKG_TARGET_IS_WINDOWS)
```
We prefer to use this. Could you please update it?